adafruit_display_text/scrolling_label.py

```diff
@@ -26,36 +26,42 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Display_Text.git"
 
+try:
+    from typing import Union, Optional
```

Comment:
You won't need `Union` anymore if you use `fontio.FontProtocol` either!

```suggestion
    from typing import Optional
```